File: src/main/java/at/spot/jython/testing/Car.java
package at.spot.jython.testing;

import at.spot.jython.PythonMethod;
import at.spot.jython.PythonClass;

@PythonClass(moduleName = "Car", className = "Car", constructorArgs = { String.class })
public interface Car {

	@PythonMethod
	String getNumberPlate();
}

File: src/test/java/at/spot/jython/Car.py
class Car():
	numberPlate = None

	def __init__(self, numberPlate = None):
		self.numberPlate = numberPlate
	
	def getNumberPlate(self):
		return self.numberPlate

	def setNumberPlate(self, numberPlate):
		self.numberPlate = numberPlate
		